soroban-sdk/src/ledger.rs

```diff
@@ -0,0 +1,47 @@
+use crate::{
+    env::internal::{self, RawValConvertible},
+    Bytes, Env,
+};
+
+#[derive(Clone)]
+pub struct Ledger(Env);
+
+impl Ledger {
+    #[inline(always)]
+    pub(crate) fn env(&self) -> &Env {
+        &self.0
+    }
+
+    #[inline(always)]
+    pub(crate) fn new(env: &Env) -> Ledger {
+        Ledger(env.clone())
+    }
+
+    /// returns the current ledger version
+    pub fn get_ledger_version(&self) -> u32 {
```

Comment:
Let's follow the patterns of other docs in the SDK, by writing sentences. For example:
https://github.com/stellar/rs-soroban-sdk/blob/bba4039966263f609ceefbf5b30a43361eecaa01/soroban-sdk/src/contract_data.rs#L79-L90

I think we should skip repeating `ledger` in the function name given it is a function on the `Ledger` type.

I think we should add the term protocol too. This isn't the version of the ledger, it's the version of the protocol that this ledger was created with. 

```suggestion
    /// Returns the version of the protocol that the ledger created with.
    pub fn protocol_version(&self) -> u32 {
```

If any of these functions panic we should add a `### Panics` section as well, which is the convention. As far as I can tell these functions will never panic, but if they do can you add sections stating when they do?

Reply:
It doesn't look like these functions panic.